src/test/shared/sam/debugger/awsSamDebugConfigurationProvider.test.ts

```diff
@@ -1,173 +0,0 @@
-/*!
```

Comment:
this file is redundant with `src/test/shared/sam/debugger/samDebugConfigProvider.test.ts`